core/app/modules/kit-library/assets/js/e-component.js

```diff
@@ -0,0 +1,76 @@
+export default class EComponent extends $e.modules.ComponentBase {
```

Comment:
What is it? :)

Reply:
This file allows us to send the fake events through the web-cli, all the commands are registered in this file.